feel-repl.sc

```diff
@@ -7,18 +7,40 @@ import $ivy.`org.apache.logging.log4j:log4j-slf4j-impl:2.14.0`, org.apache.loggi
 Configurator.setRootLevel(Level.WARN)
 
 // initialize the FEEL engine
-val feelEngine = new FeelEngine() 
+val feelEngine = new FeelEngine()
 
 // define a shortcut function for evaluation
 def feel(expression: String, context: Map[String, Any] = Map.empty) {
   feelEngine.evalExpression(expression, context) match {
     case Right(result) => println(fansi.Color.LightGreen(s"> $result"))
-    case Left(failure) => println(fansi.Color.LightRed(s"> $failure"))  
+    case Left(failure) => println(fansi.Color.LightRed(s"> $failure"))
   }
 }
 
+def feel(expression: String, jsonContext: String) {
+  val json = ujson.read(jsonContext) match {
+    case ujson.Obj(objValue) => {
+      val context = objValue.map { case (key, value) => key -> unpackJson(value)}.toMap
+      feel(expression, context)
+    }
+    case otherValue => println(fansi.Color.LightRed(s"> Expected a JSON object as variables but found '$otherValue'"))
+  }
+}
+
+def unpackJson(json: ujson.Value): Any = {
+  json match {
+    case ujson.Obj(objValue) => objValue.map { case (key, value) => key -> unpackJson(value)}.toMap
+    case ujson.Arr(arrValue) => arrValue.map(unpackJson).toList
+    case primitiveValue => primitiveValue.value
+  }
+}
+
+// print on loading the script
 println(fansi.Color.LightBlue("===== FEEL Engine REPL ======"))
 println(fansi.Color.LightBlue("""> Evaluate FEEL expressions using 'feel("1 + 3")'"""))
 println(fansi.Color.LightBlue("""> Provide variables using 'feel("x + 3", Map("x" -> 2))'"""))
```

Comment:
@saig0 I believe you should remove L41 (sorry if that wasn't clear in the previous comment)

Reply:
True. I oversaw it when viewing the PR on my phone :see_no_evil: 
I removed the line :wink: